### PR TITLE
[Fix] fix on stubs and usage of all process

### PR DESCRIPTION
**Pull request by Mxrcon**
# Fix on stubs and usage of all process

Hey @abhi18av I'd like to implement the usage of all process, fix some stubs that didn't had the right output and add echo "" instruction to them, this fix will make the workflow completely usable, hope this changes will be useful for your work.

## PR content
* fix on stubs
* echo "" to all stubs
* Implementation of all process

## Ending
I'm at your disposal to make any changes that you feel necessary, or if you have any doubt or critic, contact me.

Hope we have a merge!
Regards, Davi

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -19,7 +19,7 @@ workflow {
     gatk38_jar_ch = Channel.value(java.nio.file.Paths.get("$params.gatk38_jar"))
     env_user_ch = Channel.value("root")
 
-//   FASTQC_UNTRIMMED(reads_ch) // DONE
+    FASTQC_UNTRIMMED(reads_ch) // DONE
```
Comment (abhi18av) [inline main.nf]:
```suggestion
    FASTQC_UNTRIMMED(reads_ch)
```

File: modules/spotyping/spotyping.nf [hunk 2]
```diff
@@ -24,6 +24,7 @@ process SPOTYPING {
     """
 
     stub:
+    genomeReadToBeAnalyzed = params.R2 ? genomeReads[1] : genomeReads[0]
```
Comment (abhi18av) [inline modules/spotyping/spotyping.nf:27]: Good catch! 👍

File: modules/trimmomatic/trimmomatic.nf [hunk 3]
```diff
@@ -62,7 +62,7 @@ process TRIMMOMATIC {
     LEADING:3 TRAILING:3 SLIDINGWINDOW:4:20 MINLEN:36"
     
     touch ${genomeName}_R1.p.fastq.gz
-    touch ${genomeName}_R2.p.fastq.gzcontainer 'quay.io/biocontainers/trimmomatic:0.35--6'
```
Comment (abhi18av) [inline modules/trimmomatic/trimmomatic.nf:65]: Whoa, can't believe that this got overlooked in the last review 😅

**Review by abhi18av — APPROVED** [review]
Thanks mate, added a minor suggestion.

File: modules/trimmomatic/trimmomatic.nf (hunk 3 above)
Comment (Mxrcon) [reply modules/trimmomatic/trimmomatic.nf:65]: Me neither, I tried to read again but haven't saw that, at least is a quick-fix.

File: modules/spotyping/spotyping.nf (hunk 2 above)
Comment (Mxrcon) [reply modules/spotyping/spotyping.nf:27]: :rocket: Thanks :)

**Pull request merged**